app/src/main/kotlin/com/softteco/template/utils/UIExtension.kt

```diff
@@ -0,0 +1,16 @@
+package com.softteco.template.utils
+
+import android.graphics.Color
+import com.softteco.template.Constants
+import java.util.Random
+
+fun generateRandomColor(): Int {
+    return Random().let {
```

Comment:
There is already "generateRandomColor()" in utils/BluetoothExtension.kt

Reply:
A duplicate of this function has been removed utils/BluetoothExtension.kt.